libs/libfstrm/Makefile

```diff
@@ -0,0 +1,64 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libfstrm
+PKG_RELEASE=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/farsightsec/fstrm.git
```

Comment:
You should use tarball instead of git whenever it's possible ( probably https://dl.farsightsecurity.com/dist/fstrm/fstrm-0.5.0.tar.gz )

Reply:
Thanks! I got a bit lost in the GitHub interface but found the link myself eventually. I've made the changes to download straight from a tarball.